Add the debian janitor repositories

jelmer opened the pull request:
These are from the Debian Janitor project to automatically pull in new upstream releases and snapshots.

They're essentially overlay repositories, like e.g. Debian experimental, and do not contain the full set of packages but are intended to be used on top of e.g. unstable.

AMDmi3 commented:
https://janitor.debian.net/dists/fresh-releases/contrib/source/Sources.gz → 404
https://janitor.debian.net/dists/fresh-snapshots/contrib/source/Sources.gz → 404

jelmer commented:
> https://janitor.debian.net/dists/fresh-releases/contrib/source/Sources.gz → 404 https://janitor.debian.net/dists/fresh-snapshots/contrib/source/Sources.gz → 404

Fixed; the Janitor just publishes "main" as a compoment, so I've dropped contrib and non-free.

AMDmi3 commented:
These lack `packagelinks` at least to package sources.

jelmer commented:
> These lack `packagelinks` at least to package sources.

Added.

jelmer commented:
It looks like this commit made its way in, so I'll close this PR. Thanks, @AMDmi3 !
